web/src/pages/BlockList/index.css.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+
+export const BlockListPanel = styled.div`
+  width: 100%;
+  margin-top: ${(props: { width: number }) => (150 * props.width) / 1920}px;
+  margin-bottom: ${(props: { width: number }) => (200 * props.width) / 1920}px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const ContentTitle = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  div {
+    font-size: 50px;
+    color: black;
+    margin: 0 auto;
+  }
+
+  span {
+    background: #46ab81;
+    height: 4px;
+    width: 197px;
+    margin: 0 auto;
+  }
+`
+
+export const ContentTable = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 58px;
+`
+
+export const TableTitleRow = styled.tr`
+  background: rgb(75, 188, 142);
+  display: flex;
+  flex-direction: row;
+
+  th {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 240px;
+    height: ${(props: { width: number }) => (78 * props.width) / 1920}px;
```

Comment:
recommend to keep the height here the original 78; it was not an image fullscreen

Reply:
OK